common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -229,6 +231,7 @@ def setUp(self):
         with patch.dict(os.environ, {'SELENIUM_BROWSER': browser}):
             super().setUp()
 
+    @pytest.mark.skip(reason='This test fails when using the new courseware MFE.')
```

Comment:
I don't have any idea why this is failing. Is this test run against a particular course? Is the .srt file missing? Is this test failing for everyone else?

Reply:
I figure it is because the test expects that the video block is rendered within the legacy courseware frontend. Since this PR makes the micro-frontend the default for courseware, which is served from a different domain and has a different page structure, the test probably fails to load the page entirely, even if the video block itself would behave identically in the MFE experience.

Reply:
Yes - both tests failed because the check to wait on the page to be loaded timed out. With the different page structure, I strongly suspect the definition of what indicates a page is fully loaded has changed and needs to be updated.

Reply:
I suspect we won't even be able to run bokchoy a11y tests against the new MFE pages, since that'd involve spinning up a working frontend-app-learning as part of the PR build.

As teams continue to switch features over to be MFE-by-default, I expect we'll end up gradually deleting all the bokchoy a11y tests.